### PR TITLE
Fix flake8 failures

### PR DESCRIPTION
**Pull request by ivanpauno**
:point_up:

According to [ros2 style guide](https://index.ros.org/doc/ros2/Contributing/Code-Style-Language-Versions/#id4), "hanging indents" are preferred over "continuation lines". So, I also enforced that style (which flake8 doesn't enforce).

### CODE REVIEW

File: gazebo_ros/launch/gzclient.launch.py [hunk 1]
```diff
@@ -44,33 +45,43 @@ def generate_launch_description():
...
-
-    prefix = PythonExpression(['"gdb -ex run --args" if "true" == "',
-                                     LaunchConfiguration('gdb'),
-                                     '"else "valgrind" if "true" == "',
-                                     LaunchConfiguration('valgrind'),
-                                     '"else ""']),
+    env = {
+        'GAZEBO_MODEL_PATH': model,
+        'GAZEBO_PLUGIN_PATH': plugin,
+        'GAZEBO_RESOURCE_PATH': media}
```
Comment (chapulina) [inline gazebo_ros/launch/gzclient.launch.py]:
```suggestion
        'GAZEBO_RESOURCE_PATH': media
    }
```

**Review by chapulina — APPROVED** [review]
Thanks for the cleanup! I just have one suggestion. Also, I wonder if all the `)` should also be on a new line. The guidelines say that "_The closing brace/bracket/parenthesis on multiline constructs may either line up under the first non-whitespace character of the last line of list._"

It would be great if the linter would catch these.

File: gazebo_ros/launch/gzclient.launch.py (hunk 1 above)
Comment (ivanpauno) [reply gazebo_ros/launch/gzclient.launch.py]: Done in https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1110/commits/c18079fd9218ad93718fe38e0b909686e5d81a4d.

**Comment by ivanpauno** [comment]
> Thanks for the cleanup! I just have one suggestion. Also, I wonder if all the `)` should also be on a new line. The guidelines say that "_The closing brace/bracket/parenthesis on multiline constructs may either line up under the first non-whitespace character of the last line of list._"

I moved all closing braces/parentheses to a new line in multi-line statements.
 
> It would be great if the linter would catch these.

I'm not sure if flake8 can be configured ....

**Pull request merged**